process_report/process_report.py

```diff
@@ -221,12 +223,21 @@ def main():
     timed_projects_list = timed_projects(args.timed_projects_file, invoice_month)
     print("The following timed-projects will not be billed for this period: ")
     print(timed_projects_list)
-
     projects = list(set(projects + timed_projects_list))
 
     merged_dataframe = add_institution(merged_dataframe)
-    export_lenovo(merged_dataframe, args.Lenovo_file)
-    remove_billables(merged_dataframe, pi, projects, args.nonbillable_file)
+    lenovo_inv = lenovo_invoice.LenovoInvoice(
+        args.Lenovo_file, invoice_month, merged_dataframe.copy()
```

Comment:
For both these calls to constructors for the invoices I would advise you use named parameters. This because it's hard to trace the order of arguments in the subclasses without the lack of a `__init__()` function to look at, since this order is dependent on the order of definition of attributes of the dataclass. 

```suggestion
    lenovo_inv = lenovo_invoice.LenovoInvoice(
         name=args.Lenovo_file, invoice_month=invoice_month, data=merged_dataframe.copy()
         )
```